test/unit/qme/map/measure_calculation_job_test.rb

```diff
@@ -48,8 +48,7 @@ def test_perform_recalculate
   def test_perform_no_recalculate
     options = {
       'measure_id' => '2E679CD2-3FEC-4A75-A75A-61403E5EFEE8',
-      'effective_date' => Time.gm(2011, 1, 15).to_i,
-      'status.state' => 'complete'
```

Comment:
yeah @oniofchaos  i just doubled checked this test on the 'bump_mongoid' branch again. Quality report is created with the status 'unknown' so it doesn't make a difference if the state is specified to be 'complete' or 'completed.' Furthermore, i think it should be 'completed' and not 'complete' based off of https://github.com/q-centrix/quality-measure-engine/blob/bump_mongoid/lib/qme/quality_report.rb#LC125 and other places where this status is called.